### PR TITLE
Add process stats

### PR DESCRIPTION
**Pull request by andrewvc**
This adds process stats.

Also includes 'pry' as a dev dependency for easier debugging.
This also raises the language level to 1.7 so we can support `<>` syntax.

### CODE REVIEW

File: jruby-ext/pom.xml [hunk 1]
```diff
@@ -7,6 +7,18 @@
     <groupId>com.purbon.jruby</groupId>
     <artifactId>monitor</artifactId>
     <version>1.0</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.7</source>
```
Comment (suyograo) [inline jruby-ext/pom.xml]: this should be 1.8. We don't support 1.7 in LS 5.x

**Comment by andrewvc** [comment]
@suyograo I just bumped the version to JDK8 only.

File: jruby-ext/src/main/java/com/purbon/jrmonitor/monitors/ProcessMonitor.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        }
+
+        public short scaleLoadToPercent(double load) {
+            if (osMxBean instanceof UnixOperatingSystemMXBean) {
+                UnixOperatingSystemMXBean unixOsBean = (UnixOperatingSystemMXBean) osMxBean;
+
+                if (load >= 0) {
+                    return (short) (load * 100);
+                } else {
+                    return -1;
```
Comment (suyograo) [inline jruby-ext/src/main/java/com/purbon/jrmonitor/monitors/ProcessMonitor.java:73]: can we use a constant here?

Comment (andrewvc) [inline jruby-ext/src/main/java/com/purbon/jrmonitor/monitors/ProcessMonitor.java:73]: For -1? Why?

Comment (andrewvc) [inline jruby-ext/src/main/java/com/purbon/jrmonitor/monitors/ProcessMonitor.java:73]: FWIW that code is partially lifted from the es source code.

Comment (suyograo) [inline jruby-ext/src/main/java/com/purbon/jrmonitor/monitors/ProcessMonitor.java:73]: Ok, nm. I misread it. I thought we were sending `-1` for non existing stats.

**Comment by suyograo** [comment]
LGTM

File: .travis.yml [hunk 3]
```diff
@@ -1,9 +1,6 @@
 language: ruby
```
Comment (suyograo) [inline .travis.yml]: jruby?

Comment (suyograo) [inline .travis.yml]: never mind. its ruby

**Pull request closed without merging**